charts/kubernetes/templates/playbooks/playbooks.yaml

```diff
@@ -165,11 +165,113 @@ spec:
           kubectl create rolebinding "playbook-$(.run.id)"
           --namespace=$(.config.name)
           --user=$(.params.user)
-          --role=$(.params.role)
+          --role=$(.params.role.name)
     - name: Clear Rolebinding
       if: "success()"
       delay: ".params.delay"
       exec:
         script: |
           kubectl delete rolebinding "playbook-$(.run.id)" --namespace=$(.config.name)
 {{- end}}
+
+{{- if and .Values.playbooks .Values.playbooks.requestNamespaceAccessGitops }}
```

Comment:
This should be gated on playbooks.enabled and playbooks.edit_kubernetes_manifests.enabled

Reply:
Can we also move the playbooks to 1 per file ?